widgetssdk/src/main/java/com/glia/widgets/engagement/States.kt

```diff
@@ -12,19 +12,17 @@ internal sealed interface State {
     data object QueueUnstaffed : State
     data object UnexpectedErrorHappened : State
     data object QueueingCanceled : State
-    data object StartedOmniCore : State
-    data object StartedCallVisualizer : State
-    data object FinishedOmniCore : State
-    data object FinishedCallVisualizer : State
+    data class EngagementStarted(val type: EngagementType) : EngagementState(type)
+    data class EngagementEnded(val type: EngagementType, val isEndedByVisitor: Boolean, val onEnd: ActionOnEnd) : EngagementState(type)
     data object TransferredToSecureConversation : State
-    data class Update(val state: EngagementState, val updateState: EngagementUpdateState) : State
+    data class Update(val state: com.glia.androidsdk.engagement.EngagementState, val updateState: EngagementUpdateState) : State
```

Comment:
Have you dropped the `import com.glia.androidsdk.engagement.EngagementState` and added `com.glia.androidsdk.engagement.` here to the code on purpose?